packages/grid/helm/syft/values.yaml

```diff
@@ -37,34 +37,57 @@ frontend:
   resources: null
 
 seaweedfs:
-  # S3 settings
+  # Master
+  master:
+    # Seaweed Volume Size in MB set on master. This is not PVC.
+    volumeSizeMB: 1000
```

Comment:
Q: 1. This the maximum capacity of all the Volume servers overall?
      2. Can we control the number of volume servers spinned up?